lib/driver.js

```diff
@@ -771,12 +769,63 @@ class XCUITestDriver extends BaseDriver {
     }
     log.debug(`Setting initial orientation to '${orientation}'`);
     try {
-      await this.proxyCommand('/orientation', 'POST', {orientation});
+      await this._executeCommandWithTimeout(B.promisify(this.proxyCommand), 'setScreenOrientation', '/orientation', 'POST', {orientation});
```

Comment:
yep, might be a good idea. Though, there is one important question - how can I map endpoint name to a command name (do a reverse lookup to what is happening in **executeCommand**)? This is needed to be able to select a timeout for the particular command in **this.opts.commandTimeouts** if it is represented as dictionary.